topics/climate/tutorials/ocean-variables/tutorial.md

```diff
@@ -96,7 +91,7 @@ Once you finish your work on ODV, if you want to retrieve data and/or the entire
 >    - *"Select if you are using an ODV collection in a zip folder or if you have your own raw data"*: `The data you are using are an ODV collection in a zip folder`
 >    - *"ODV collection in a zip folder."*: `Eutrophication_Med_profiles_2022_unrestricted_SNAPSHOT_2023-10-24T16-39-44.zip`
```

Comment:
where does this file come from? Is it in a zenodo somewhere or ?

Reply:
depending on the size, zenodo is also good! (We try and avoid having too-large files in the GTN where possible.)

Reply:
It's big but I loaded it in the Fair-Ease galaxy